teuthology/task/install.py

```diff
@@ -520,6 +533,50 @@ def verify_package_version(ctx, config, remote):
         )
 
 
+def _install_rh_build(ctx, remote, config):
+    """
+    Installs RH build using ceph-deploy.
+
+    TODO: split this into at least two functions.
+
+    :param ctx: the argparse.Namespace object
+    :param rpm: list of packages names to install
+    :param config: the config dict
+    """
+    pkgs=['ceph-deploy']
+    for pkg in pkgs:
+      log.info("Check if ceph-deploy is already installed on system")
```

Comment:
We have a function already in teuthology.packaging that should be able to tell you if a package is installed on a remote.  You could probably use that here. 

https://github.com/ceph/teuthology/blob/master/teuthology/packaging.py#L203